config.omero

```diff
@@ -0,0 +1,12 @@
+# By default, we enable 'minimal_webapp' only
+config append omero.web.apps '"minimal_webapp"'
+# Comment this out to enable
+# config append omero.web.apps '"react_webapp"'
+
+config set omero.web.debug True
+
+# enable CORS so e.g. react-webapp can connect to OMERO.web on different origin
```

Comment:
Is this _only_ needed for the react app?

Reply:
Yes, currently that's the only example app here that can be run in a development JavaScript-based server at localhost:3000. All the others are built and served by OMERO.web so they don't need to connect from a different origin.